### ISSUE TITLE
Notice of Copyright Violation

### ISSUE THREAD

**Issue by Iconoclasta** [kalkulusteam/klks#5]
This is an official notice of Copyright violation against the PIVX developers in which this project has been forked from. The MIT copyright license, and subsequent copyright headers in source files have been modified to explicitly exclude the PIVX developers from all copyrights.

You are hereby notified of an intention to pursue further action if this issue is not acknowledged and formally addressed with a resolution time frame commitment wherein. Failure to acknowledge and correct the violation will result in a formal DMCA take-down notice filed with GitHub Proper after no less than five business days from the date of this notice.

Explicit acknowledgement and referencing progress to maintain compliance MUST be noted in this issue's comments or referred to in any subsequent pull requests to this repository.

**Comment by Iconoclasta** [comment]
Over 3 weeks have passed and you have not solved this issue. I will be passing this notice to our legal department should you not comply within 24h.

**Comment by Iconoclasta** [comment]
Yo

**Comment by kalkulusteam** [comment]
Copyright violation on open source project? 
In all case this project is a fork of a fork of the fork of (maybe) PIVX and is not directly related to this project, we didn't modified headers voluntarly excluding PIVX.
Any "violation" you found here, is not intented to be done with the purpose of violate copyright or exclude owners from their original works.

**Comment by Fuzzbawls** [comment]
yes...copyright violation on an open source project..hope that wasn't a serious question.

Regardless of where you forked from, you are responsible for ensuring that proper copyrights are retained. Here, every `.h` or `.cpp` file that contained a PIVX copyright in the header has had that copyright replaced with `2017-2018 Kalkulus Development Team`, probably the result of a poor search/replace.

Intentional or not doesn't matter, but the problem absolutely does need to be resolved

**Comment by kalkulusteam** [comment]
As already said, was not an intentional infringement and hope you trust me
that Kalkulus replaced the name of previous project and not PIVX. In all
case, as we have maximum respect for your works and contribution that you
provide everyday, I will fix that adding the missing copyright line
crediting PIVX as is correct.
Best regards


Il gio 4 ott 2018, 07:31 Fuzzbawls <notifications@github.com> ha scritto:

**Comment by kalkulusteam** [comment]
https://github.com/kalkulusteam/klks/commit/7c24a82774c274b7acceed95c44bec3e84235289
Can we consider this enough to close the issue?
